### DISCUSSION TITLE
Dongle serial number contains letters...

### DISCUSSION THREAD

**Discussion by paulbloggs** [davidrapan/ha-solarman#139] [Q&A] [answered]
Hi,
Probably I am doing something silly.
I have just installed a Sunsynk 5kW inverter and am trying to install solarman on my home assistant.
The configuration asks for a serial number which I think is supposed to be the dongle serial number?
My serial number has letters in it and the configuration will not accept it.
How do I fix this?
Regards
Paul

**Comment by davidrapan** [comment, marked as answer]
I'm afraid that proly means that you have an "older" not supported version...

More 'bout it for example here: https://github.com/jmccrohan/pysolarmanv5/issues/8

Edit: Supported is for example LSW-3 or LSE-3 from https://www.solarmanpv.com/products/data-logger/stick-logger/

**Comment by paulbloggs** [comment]
It is brand new - only 2 weeks old!

**Reply by davidrapan** [reply]
Yeah but maybe the dongle could be an older one... I'm not sure I just know that letter were used and now are not... What dongle you have?

**Reply by paulbloggs** [reply]
Hardware Version:AEW2-0005-03
Software Version:4790123B24R

Does that help?
Thanks for the reply

**Reply by davidrapan** [reply]
Yes this is one of the older devices, you will have to replace your dongle if you want to use this integration.

**Reply by paulbloggs** [reply]
What versions should i look for?

**Reply by davidrapan** [reply]
LSW-3 from https://www.solarmanpv.com/products/data-logger/stick-logger/ for example.

**Reply by paulbloggs** [reply]
Thanks. 
Is there a way of using a raspberry like I did with my Victron?
Or ethernet?

**Comment by davidrapan** [comment]
> Is there a way of using a raspberry like I did with my Victron?

There is but it's not user-friendly for regular users and you have to use sunsynk addon for HA instead of this integration.

> Or ethernet?

On the site I gave you in previous post is also ethernet version LSE-3 which is also supported by this integration.

😉

**Reply by paulbloggs** [reply]
Thanks David,
I will have a look at the sunsynk integration - spent all my money on the inverter and battery!!
